### PR TITLE
Instagram messages

### LINKED ISSUE #24: Add support for Instagram messages
rip instalysis 2021-2021

### PR DESCRIPTION
**Pull request by miskfi**
Added support for analyzing Instagram messages (closes #24). The user chooses the message source (Instagram or Messenger) in the beginning and the program then runs the analysis for the selected source type. The output HTMLs are now stored in folders based on the message source.

The template aligning for Instagram needs to be fixed, but I'll leave that to @stepva.

### CODE REVIEW

File: chatalysis/chats/chat.py [hunk 1]
```diff
@@ -29,7 +29,9 @@ def first_message(self) -> dict:
         pass
 
 
-class FacebookMessengerChat(Chat):
+class FacebookChat(Chat):
```
Comment (stepva) [inline chatalysis/chats/chat.py:32]: MessengerChat is better imo but idk

**Comment by miskfi** [comment]
Rebased the Instagram on top of master, everything seems to be working.

**Comment by stepva** [comment]
cool
@stepva adjust template (remove stickers/files)
@miskfi adjust GUI (larger buttons in main menu, i'm too old)
mute the warning message about stickers and files for Instagram (keep for messenger for now, unless it is too complicated?) (don't delete the code, just comment it out)

**Review by stepva — APPROVED** [review]
stonks

**Pull request merged**